mythril/laser/ethereum/svm.py

```diff
@@ -259,7 +259,7 @@ def _execute_transactions(self, address):
             )
             if func_hashes:
                 func_hashes = [
-                    bytes.fromhex(hex(func_hash)[2:]) for func_hash in func_hashes
+                    bytes.fromhex(func_hash[2:]) for func_hash in func_hashes
```

Comment:
`literal_eval` in line 619 in cli.py changes the hex to integer. Hence we will have to re-hex it. Hence this change will not work with the current cli.py file.

Reply:
> `literal_eval` in line 619 in cli.py changes the hex to integer. Hence we will have to re-hex it. Hence this change will not work with the current cli.py file.

The problem is that re-hex an integer to a string will miss the leading zeros which causes the error of `non-hexadecimal number found` in bytes.fromhex().
For example:
```python3
original_hash = '0x0b95d228'
hash_integer = int(original_hash, 16)
rehexed_hash = hex(hash_integer) # 0xb95d228, missing the leading zeros compared to original_hash
bytes.fromhex(rehexed_hash[2:]) # ValueError: non-hexadecimal number found in fromhex() arg at position 7
```
Also, I noticed that the type hint of `args.transaction_sequences` in [line 19 of support_args.py](https://github.com/ConsenSys/mythril/blob/e6c140fd891243955f6c64685a947ea6e6353f0b/mythril/support/support_args.py#L19) is `List[List[str]]`. And I tried to run the myth cli with something like `--transaction-sequences '[["0x0b95d228"]]'` (quoting the function hash), which causes the error of `'str' object cannot be interpreted as an integer` due to the re-hex operation. So I removed the re-hex here.

Reply:
You do not need to use '' when using --transaction-sequence. Something like `--transaction-sequences [["0x89b8ae9b"],["0xaaaaaaaa"],[]]` works

Reply:
Yep, this works, because the shell (bash I use) expand `[["0x89b8ae9b"],["0xaaaaaaaa"],[]]` to `[[0x89b8ae9b],[0xaaaaaaaa],[]]` before passing to myth. I add a statement `print(args.transaction_sequences)` before do `literal_eval` to check the value of `args.transaction_sequences`, and it prints `[[0x89b8ae9b],[0xaaaaaaaa],[]]`.

Reply:
I think this issue has already been sloved in the newest commit of the develop branch, so I remove my fix code